conda-forge.yml

```
@@ -7,6 +7,8 @@ provider:
   linux_aarch64: default
   linux_ppc64le: default
   win: azure
+azure:
+  upload_packages: False
```

```suggestion
azure:
  store_build_artifacts: True
```
Sorry, I had originally posted the wrong code but then edited the comment but I guess you saw it before. This should do the trick.

---

I wrote `False` by mistake anyways, so it would have failed either way :)